independent-publisher-connectors/ORB Intelligence/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4731 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ORB Database",
+    "version": "v3",
+    "description": "Orb provides with a framework of API calls to use the Orb database for various use cases. The framework comprises five calls, Match, Fetch, Search, Lookalike and Corporate Tree.",
+    "contact": {
+      "name": "Aaryan Arora, Ankita Singh",
+      "email": "aaryan2134@gmail.com"
+    }
+  },
+  "host": "api.orb-intelligence.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/3/match/": {
+      "get": {
+        "parameters": [
+          {
+            "in": "query",
+            "name": "api_key",
+            "required": false,
+            "type": "string",
+            "default": "c66c5dad-395c-4ec6-afdf-7b78eb94166a",

```

Comment:
@aaryan2134 Please remove the secret-look-a-like values from all the connector artifacts as we receive security alerts from our internal security monitoring tools. And please let us know why you are supplying API Key for each operation where your connector has an authentication method as "API Key".  If it was added as part of testing, please remove them.

Reply:
Done